fix: add whitespace collapsing and text stripping by default

roedoejet opened the pull request:
<!-- PR template: please provide enough information to guide your reviewers.
Please read Contributing.md before submitting a PR.  -->

### PR Goal? <!-- Explain the main objective of this PR. -->

Whitespace collapsing wasn't being applied by the wizard, only the chosen cleaners. This changes that.

### Fixes? <!-- List any issues this PR fixes, e.g. Fixes #42, Fixes #324 -->



### Feedback sought? <!-- What should reviewers focus on in particular? -->

sanity, code check

### Priority? <!-- How soon would you like this PR reviewed, does it block other work? -->

medium

### Tests added? <!-- Make sure your PR includes automated tests for your changes. -->

✅ 

### How to test? <!-- Explain how reviewers should test this PR. -->



### Confidence? <!-- How confident are you that these changes are ready to merge? -->

medium-high

### Version change? <!-- Do you think this PR should trigger a Major (Breaking Change)/Minor (New Feature)/patch (refactor/bug fix) version change? -->

no

### Related PRs? <!-- List any submodule PRs required for this PR to make sense. -->

#515 and #516 

<!-- Add any other relevant information here -->

wiitt reviewed (commented):
Is it an expected behaviour that, if no text normalization is applied, character list remains duplicates of empty symbols?
For example, creating a project from `/sgile/data/MohawkCorpus/am_corpus` without text normalization results in
```
moh_characters: ['', '', (, ), '0', '1', '3', '8', a, c, d, e, g, h, i, k, l, m,
    n, o, p, r, s, t, u, v, w, x, y, z, '', à, á, è, é, ì, í, ò, ó]
```
where you can find three instances of ''.

roedoejet commented:
> Is it an expected behaviour that, if no text normalization is applied, the character list remains duplicates of empty symbols? For example, creating a project from `/sgile/data/MohawkCorpus/am_corpus` without text normalization results in
> 
> ```
> moh_characters: ['', '', (, ), '0', '1', '3', '8', a, c, d, e, g, h, i, k, l, m,
>     n, o, p, r, s, t, u, v, w, x, y, z, '', à, á, è, é, ì, í, ò, ó]
> ```
> 
> where you can find three instances of ''.
> 
> I also noticed that changes from #515 are highlighted as changes of this PR as well.

great find @wiitt - no, this is not intended behaviour. It was an unnecessary `if response:` condition that was preventing it from applying. I've fixed this and added a test to catch this in the future.
